src/EnergyPlus/BoilerSteam.cc

```diff
@@ -394,7 +394,7 @@ namespace BoilerSteam {
                 ErrorsFound = true;
             }
 
-            if (rNumericArgs(5) == 0.0) {
+            if (rNumericArgs(5) < 0.0) {
```

Comment:
IDD shows allowed boiler min flow rate = 0, defect file halted here.

Reply:
Makes sense that you should only be warning if it's less than zero.